store/presenters/presenters.go

```diff
@@ -142,8 +144,10 @@ func NewConfigWhitelist(config store.Config) ConfigWhitelist {
 		MinimumContractPayment:   &config.MinimumContractPayment,
 		MinimumRequestExpiration: config.MinimumRequestExpiration,
 		OracleContractAddress:    config.OracleContractAddress,
-		Port:    config.Port,
-		RootDir: config.RootDir,
+		Port:             config.Port,
+		RootDir:          config.RootDir,
+		SessionTimeout:   config.SessionTimeout,
```

Comment:
Are `SessionTimeout` & `ReaperExpiration` from another PR?

Reply:
Yes, but they weren't added to presenters.